sequelize/seeders/memeolist-example-postgres.js

```diff
@@ -40,6 +40,8 @@ const resolvers = [
     GraphQLSchemaId: 1,
     requestMapping: 'SELECT * FROM meme ORDER BY id DESC',
     responseMapping: '{{ toJSON context.result }}',
+    preHook: 'http://www.google.com',
```

Comment:
I will setup this on mutation instead of query.
Additionally we can use environment variable here in order to override that in example. see that request is made

Reply:
This is only for test and going to be removed